src/components/checkboxes/_checkbox.scss

```
@@ -19,7 +18,7 @@ $checkbox-label-vertical-padding: 0.75rem;
     height: $checkbox-input-width;
     left: $checkbox-padding;
     position: absolute;
-    top: $checkbox-padding + 0.15rem;
+    top: $checkbox-padding + 2px;
```

Vertical alignment not quite right.
 ```suggestion
    top: $checkbox-padding + 3px;
```